modern_treasury/modern_treasury.py

```diff
@@ -238,4 +241,19 @@ def list_incoming_payment_detail(self, virtual_account_id: str = None) -> List[O
             response = self._get(url=LIST_INCOMING_PAYMENT_DETAIL_URL, params=querystring)
             return [IncomingPaymentDetailResponse(payment_order) for payment_order in response]
         except:
-            return []
\ No newline at end of file
+            return []
+    
+    def list_connections(self, vendor_customer_id: Optional[str] = None, entity: Optional[str] = None) -> List[ConnectionResponse]:
```

Comment:
do we need the idempotency key for a GET?

Reply:
Also please add some tests 🙇

Reply:
> do we need the idempotency key for a GET?

Nope, we don't need idempotency key for GET requests